spec/factories/books.rb

```diff
@@ -0,0 +1,10 @@
+FactoryBot.define do
+  factory :book do
+    genre     { Faker::Book.genre }
+    author    { Faker::Name.name }
+    image     "image.jpg"
+    title     { Faker::Book.title }
+    publisher { Faker::Book.publisher }
+    year      { Faker::Number.between(1800, 2010) }
+  end
+end
```

Comment:
If you use Visual Studio Code add this line to your preferences file `"files.insertFinalNewline": true` to avoid this happening.